src/gazebo_geotagged_images_plugin.cpp

```diff
@@ -468,7 +481,8 @@ bool GeotaggedImagesPlugin::_init_udp(sdf::ElementPtr sdf) {
     _fds[0].events = POLLIN;
     mavlink_status_t* chan_state = mavlink_get_channel_status(MAVLINK_COMM_1);
     chan_state->flags &= ~(MAVLINK_STATUS_FLAG_OUT_MAVLINK1);
-    gzmsg << "Camera on udp port 14530\n";
+    std::string debug_msg = "[Camera manager plugin]: Camera on udp port " + std::to_string(_camPort) + "\n";
+    gzmsg << debug_msg;
```

Comment:
No need for an intermediate variable
```suggestion
    gzmsg << "[Camera manager plugin]: Camera on udp port " + std::to_string(_camPort) + "\n";
```